secant/SecantApp.swift

```diff
@@ -6,24 +6,31 @@
 //
 
 import SwiftUI
+import ComposableArchitecture
+
+final class AppDelegate: NSObject, UIApplicationDelegate {
+    var appStore: AppStore = .placeholder
+    lazy var appViewStore = ViewStore(
+        appStore.scope(state: { _ in () }),
```

Comment:
I _think_ you can call stateless here instead of using this scope
https://github.com/pointfreeco/swift-composable-architecture/blob/2828dc44f6e3f81d84bcaba72c1ab1c0121d66f6/Sources/ComposableArchitecture/Store.swift#L395

Reply:
this is really great, thanks for letting me know about stateless as well as actionless, fixed